scripts/h5toms.py

```diff
@@ -60,10 +60,8 @@
                   help="More verbose progress information")
 parser.add_option("-w", "--stop-w", action="store_true", default=False,
                   help="Use W term to stop fringes for each baseline")
-parser.add_option("-x", "--HH", action="store_true", default=False,
-                  help="Produce a Stokes I MeasurementSet using only HH")
-parser.add_option("-y", "--VV", action="store_true", default=False,
-                  help="Produce a Stokes I MeasurementSet using only VV")
+parser.add_option("-p", "--pols-to-use", default=None,
+                  help="Select polarisation products to include in MS from HH,VV,HV,VH, default is all available from HH,VV")
```

Comment:
Is this a comma-separated list of polarisation products, i.e. can you go "HV,VH" if you so wished?